### ISSUE TITLE
Server does not return valid JSON

### ISSUE THREAD

**Issue by andreaferretti** [spark-jobserver/spark-jobserver#25]
Many responses from the server are returned with content-type `application/json`, but only contain OK (without quotes) as the body.

This makes many clients fail, as OK is not valid JSON. It would be enough to either
- return "OK" with quotes, or
- change the content-type to text/plain for those responses

**Comment by xjrk58** [comment]
Maybe better to return valid JSON by replacing all 
ctx.complete("OK") with
ctx.complete(StatusCodes.OK, Map(StatusKey -> "OK"))

**Comment by noorul** [comment]
I don't think we need status for most of the operations. In REST world we should actually be using HTTP status codes.

**Comment by velvia** [comment]
There’s a difference between status codes for a job and the status of the HTTP request itself, which is what the HTTP codes are for.

For example:

HTTP 200 - “OK”   - request succeeded and job succeeded
HTTP 200 - “ERROR”   - request succeeded for info, job errored out
HTTP 500 -   something wrong with request (wrong job ID - actually that would be 400) or job server

I think the distinction is important.

**Comment by CalebFenton** [comment]
HTTP 200 with a status of ERROR may also mean akka had a timeout (e.g. akka.pattern.AskTimeoutException). It's a bit overloaded.
